packages/react-components/package.json

```diff
@@ -42,10 +43,12 @@
     "@internal/northstar-wrapper": "1.5.1-beta.3",
     "copy-to-clipboard": "~3.3.1",
     "html-to-react": "~1.5.0",
+    "lodash": "^4.17.21",
     "react-aria-live": "^2.0.5",
     "react-linkify": "^1.0.0-alpha",
-    "uuid": "^8.1.0",
-    "react-use-draggable-scroll": "~0.4.7"
+    "react-use-draggable-scroll": "~0.4.7",
+    "textarea-caret-ts": "^4.1.1",
```

Comment:
Looks like this package do have its ts definition:
https://www.npmjs.com/package/@types/textarea-caret
Is it possible to reuse this? 
textarea-caret-ts looks like a fork that hasn't been updated since 4 yrs ago

Reply:
textarea-caret-ts is a TS package and has typings already.

Reply:
Another concern is download number - [@types/textarea-caret](https://www.npmjs.com/package/@types/textarea-caret) has 10k+ download per week, and [textarea-caret](https://www.npmjs.com/package/textarea-caret) has 150k+ download
And the textarea-caret-ts is only a fork with 900+ download a week.
If it is possible, it's preferred to use more popular package and original package + types definition, cause it might get more validation through more people and easier to get things fixed

Reply:
But lemme know if there is some APIs specifically in textarea-caret-ts, so doing a package switch will be time consuming

Reply:
While the JS package has more downloads, this is likely due to the fact that the fork is more recent and is TS oriented. JS is still more prevalent than TS so I would expect numbers to be different here.
What we do know is that the package selected is working as expected and is more recently updated. I think for now, we leave this known package in place.